### DISCUSSION TITLE
[Question] How do I accept copilot extension in vscode

### DISCUSSION THREAD

**Discussion by KSH-code** [community/community#7178] [Copilot Conversations] [answered]
Hey!

I've been struggling to enable copilot.
Few months ago, I installed copilot extension. but I've not accept. then I've upgraded macOS's version to Bigsur from Mojave and also to latest vscode version.
- Version: 1.61.2 Commit: 6cba118ac49a1b88332f312a8f67186f7f3c1643

After that, I would like to enable copilot. but the error occurs.
'command not found'
<img width="1609" alt="스크린샷 2021-11-03 오후 3 51 37" src="https://user-images.githubusercontent.com/29705162/140019444-e2609cf2-878f-4ea8-8558-56ac7ab2683b.png">

- no error occurs on output.

<img width="355" alt="스크린샷 2021-11-03 오후 3 51 47" src="https://user-images.githubusercontent.com/29705162/140019447-e214c123-89cb-4f96-b553-0a96c1d18103.png">

I've used that ways
1. rm -rf ~/.vscode/extensions/github.copilot-1.6.3615/
2. uninstall end install

**Comment by davecheney** [comment]
@KSH-code please try installing the github copilot extension from the extension marketplace inside vscode.

**Reply by KSH-code** [reply]
@davecheney Thank you for quick respond.
I've tried re-installing form vscode marketplace inside vscode.
<img width="1630" alt="스크린샷 2021-11-03 오후 4 12 11" src="https://user-images.githubusercontent.com/29705162/140021132-44a70712-994d-4b34-be56-577fb349d4e0.png">

**Reply by KSH-code** [reply]
I've just reinstalling just in case. but the same as before.

**Reply by davecheney** [reply]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by KSH-code** [reply]
@davecheney 
Tunnelblick is installed on my laptop. but it is not enabled. I only enable it when it is needed.
And I'm using Starbucks secure WiFi now.

no other is using.

**Reply by KSH-code** [reply]
Now, I'm at home. so I'm not using public secure WiFi.
but the same as before.

**Reply by KSH-code** [reply]
@davecheney 
I've checked my all of configs, but there is no intercepting. I've seen my situation to many people. so I think it needs manual to debug or check configs.

**Comment by KSH-code** [comment, marked as answer]
Complete by https://jimkang.medium.com/complete-uninstall-remove-vscode-mac-5e48bef3bdec

remove all of vscode data including synced settings.
then install vscode and install copilot extension.

**Comment by LukyVj** [comment]
On my end, I didn't had to remove all vscode data, but simply manually add: 
```json
"github.copilot.enable": {
    "*": true,
    "yaml": false,
    "plaintext": false,
    "markdown": false
  },
```

to my `settings.json` in vscode, then quit the app and restart the app, and I was prompted with the Github sign-in window.
